include/odp/api/spec/ipsec.h

```diff
@@ -1277,6 +1279,7 @@ int odp_ipsec_in(const odp_packet_t pkt_in[], int num_in,
  *
  * Each input packet must have a valid value for these metadata (other metadata
  * is ignored):
+ * - IPv4 or IPv6 flag: Set packet flag according to packet contents.
```

Comment:
@psavol @JannePeltonen @muvarov @NikhilA-Linaro
In most cases application will already have knowledge about payload protocol. So it is easier to provide such knowledge to IPsec machinery. So this is basically a question to @NikhilA-Linaro, If I got it right from SEC manual, your hardware also requires providing IP version, does it not?

Reply:
I think it is overhead for the the implementations(NXP) where hardware itself retrieves IP version from the packet. if the implementation needs it, it should read from packet data itself.